concordia/admin/__init__.py

```diff
@@ -310,6 +310,13 @@ class ItemAdmin(admin.ModelAdmin):
 
     actions = (publish_item_action, unpublish_item_action)
 
+    def get_deleted_objects(self, objs, request):
+        deleted_objects = [str(obj) for obj in objs]
+        model_count = {'Items': len(deleted_objects)}
+        perms_needed = []
```

Comment:
This should probably at the least include the item modify permission requirement

Reply:
sounds good, let me see what that permission looks like.

Reply:
ok added set() to perms_needed propery